src/pool/pool_disjoint_ctl.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2025 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+*/
+
+#include "pool/pool_disjoint_internal.h"
+#include "umf/base.h"
+#include <stdio.h>
+#include <string.h>
+
+#include <ctl/ctl.h>
+#include <umf/memory_pool.h>
+#include <umf/memory_pool_ops.h>
+#include <umf/memory_provider.h>
+
+struct ctl disjoint_ctl_root;
+static UTIL_ONCE_FLAG ctl_initialized = UTIL_ONCE_FLAG_INIT;
+
+static int CTL_READ_HANDLER(name)(void *ctx, umf_ctl_query_source_t source,
+                                  void *arg, size_t size,
+                                  umf_ctl_index_utlist_t *indexes,
+                                  const char *extra_name,
+                                  umf_ctl_query_type_t queryType) {
+    (void)source, (void)indexes, (void)queryType, (void)size;
+    disjoint_pool_t *pool = (disjoint_pool_t *)ctx;
```

Comment:
please add an empty line for more readability

Reply:
Done.